Quantum-Control-Applications/Superconducting/Kerr-Cat SNAIL-mon/Use Case 2 - QuLAB - Kerr-Cat/readme.md

```diff
@@ -0,0 +1,200 @@
+(ii) wait for a cooldown-time, (iii) apply a cqr pulse and do state discrimination, (iv) wait for a variable
+delay, and (v) apply a second cqr pulse. This pulse sequence is repeated N times to get good SNR.
+
+The experimental data can be seen in Fig. 4, and by fitting it to single exponential decay we find
+$\tau_{\alpha} = 163 \mu s$ and $\tau_{-\alpha} = 164 \mu s$.
+
+![Free_evol](Figure_4_free_evolution.png)
+
+**Figure 4**, the blue and orange lines show the decoherence of |$\pm \alpha$ $>$ into the mixed state
+of equal population of the coherent states.
+
+<img alt="Free_evol_pulse" height="164" src="Fig5_free_evol_pulse_seq.png" width="399"/>
+
+**Figure 5**, cartoon of the pulse sequence for measuring the lifetime during free evolution.
+
+The main lines of QUA code that produce Fig. 4 are shown below. The QUA code that reproduces the data is `T_coh_free_decay.py`.
+
+```Python
+        with for_(tau, tau_min, tau <= tau_max, tau+dt):
+            
+            assign(flat_time,cooldown_time+8e3//4+tau) 
+            
+            play('on', 'squeeze_switch', duration=flat_time+1e3//4)
+            
+            play('ftc_rise', 'squeeze_rise')
+            align('squeeze_rise', 'squeeze_drive')
+            
+            play('cw', 'squeeze_drive', duration=flat_time)
+            
+            align('squeeze_drive', 'squeeze_fall')
+            play('ftc_fall', 'squeeze_fall')
+            
+            wait(cooldown_time, 'cqr_drive', 'resonator', 'cqr_switch','SPC_pump')
+            
+            play('on', 'cqr_switch', duration=(cqr_len//4))
+            play('cqr', 'cqr_drive')
+            play('on', 'SPC_pump', duration=(passive_len//4))                      
+            measure('passive_readout', 'resonator', None, demod.full('rotated_cos', I1, 'out1'), demod.full('rotated_sin', Q1, 'out1'))
+            
+            assign(res1,I1>I_th)
+            save(res1, res_st1)
+            save(I1, I_st1)
+            save(Q1, Q_st1)
+            
+            
+            wait(tau, 'cqr_drive', 'cqr_switch', 'resonator', 'SPC_pump')
+            
+            play('on', 'cqr_switch', duration=(cqr_len//4))
+            play('cqr', 'cqr_drive')
+            play('on', 'SPC_pump', duration=(passive_len//4))                      
+            measure('passive_readout', 'resonator', None, demod.full('rotated_cos', I2, 'out1'), demod.full('rotated_sin', Q2, 'out1'))
+            
+            assign(res2,I2>I_th)
+            
+            save(res2, res_st2)
+        
+            save(I2, I_st2)
+            save(Q2, Q_st2)
+            
+            wait(cooldown_time)
+```
+
+## Coherence times while continuous measuring
+
+Figure 6 and 7 we show the lifetimes and pulse sequence of the |$\alpha$ $>$ and |$-\alpha$ $>$ states while
+continoulsy performing measurements to the qubit. The lifetimes are smaller due to the non-perfect QNDs of
```

Comment:
Is there a reason to measure continuously from the physics standpoint or is it just to quantify the QND imperfection and show how to do it in QUA?